### DISCUSSION TITLE
Delete/Get indexed entries by type

### DISCUSSION THREAD

**Discussion by AndiLeni** [loupe-php/loupe#67] [Ideas]
Hello,

it woule be nice, if deleting all documents where an attribute matches could be batch deleted.
Also getting all documents where a type matches would be nice.

F.e. I have this document structure:
```
[
    'id' => string,
    "domain" => string,
    'type' => "sitemap" or "article",
    'title' => string,
    'content' => string,
];
```
With a cronjob I want to reindex my entries which have the type attribute "sitemap".
Bu I do not know which entries got deleted manually so I can not delete them by their id.
So I would like to dump all entries where type=sitemap and recreate them all.

Thanks and kind regards

**Comment by Toflar** [comment]
Shouldn't be too hard to implement, the tools are mostly there :) Feel to work on a PR 😎
